### DISCUSSION TITLE
How to add elastalert*' to es_index_patterns setting

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#11544] [2.4] [answered]
### Version

2.4.20

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

other (please provide detail below)

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

26

### Storage for /

200 gb

### Storage for /nsm

1 tb

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

It is a virtual machine on a esxi server, that I just installed in replacement of my 2.3 SO. Nice work guy's 

In the documentation is not clear stated where I can add the value elastalert*' to the es_index_patterns setting

[link to the documentation](https://docs.securityonion.net/en/2.4/elastalert.html?highlight=elastalert#elastalert-rules)

Can someone clearly explain to me where this value should be added? The location of the config file?

Thanks in advance

Bart

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by cm-ops** [comment]
In the GUI, select Options and toggle on show all configurations (at the top). Then `SOC > server > modules > Elastic > index` and add it there.

**Reply by sleepingbel** [reply, marked as answer]
I do not see any index page when i follow your path in my version. 
I have: soc > config > server > modules > Elastic  but no index.
![Knipsel](https://github.com/Security-Onion-Solutions/securityonion/assets/29328171/d4173cb6-efcb-4cc1-8ec0-89d08a9a294f)
I have searched in all soc config value's and did not see the relevant place to add the elastalert value

**Reply by cm-ops** [reply]
You have to toggle on Advanced view - `In the GUI, select Options and toggle on show all configurations (at the top)`

**Reply by sleepingbel** [reply]
Thx
